ganga/GangaCore/Lib/Condor/Condor.py

```diff
@@ -797,4 +797,38 @@ def timedetails(self):
     updateMonitoringInformation = \
         staticmethod(updateMonitoringInformation)
 
+    def peek(self, filename=None, command=None):
+        job = self.getJobObject()
+
```

Comment:
What happens if you call `peek` on a master job (where there is no actual job running, so nothing to `peek`). You need to check if your `job` object has subjobs, and if yes, return `False`.

Reply:
Done